deployment/deploy/103_DllrTransferWithPermit.ts

```diff
@@ -0,0 +1,62 @@
+import { DeployFunction } from "hardhat-deploy/types";
+import { DLLR, MassetManager } from "types/generated";
+import { masset } from "types/generated/artifacts/contracts";
+import { ethers, network } from "hardhat";
+import { transferOwnership } from "../../scripts/helpers/helpers";
+
+const func: DeployFunction = async (hre) => {
+    const {
+    deployments,
+    getNamedAccounts,
+    } = hre;
+
+  const { deploy, get, log } = deployments;
+  const { deployer } = await getNamedAccounts();
+  const deployedDllr = await deployments.get("DLLR");
+  let targetOwner = "";
+
```

Comment:
the script is missing upgrade part - use upgradeWithTransparentUpgradableProxy

Reply:
updated in this [commit](https://github.com/DistributedCollective/mynt/pull/17/commits/57307ba37bc62d3f003c1b59b86350579a01d092)